Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -88,6 +88,9 @@ public enum FeatureFlag: String, CaseIterable {
     /// Enable the new Upgrade Experiments
     case upgradeExperiment
 
+    ///
```

Comment:
Should we add a comment about it?

Reply:
Doh! I started the comment but dit not finish. There now.